defaults/jq/semconv.jq

```diff
@@ -104,3 +108,51 @@ def semconv_grouped_metrics($options): semconv_metrics($options) | semconv_group
 
 # Convenience function to group all metrics by their root namespace without any filtering options.
 def semconv_grouped_metrics: semconv_grouped_metrics({});
+
+#################
+# Event Functions
+#################
+
+# Extracts and processes semantic convention events based on provided options.
+# $options is an object that can contain:
+# - exclude_stability: a list of stability statuses to exclude.
+# - exclude_deprecated: a boolean to exclude deprecated events.
+# - exclude_event_namespace: a list of event domain to exclude.
+def semconv_events($options):
+    .groups
+    | map(select(.type == "event"))
+    | if ($options | has("exclude_stability")) then
+        map(select(.stability as $st | $options.exclude_stability | index($st) | not))
+      else
+        .
+      end
+    | if ($options | has("exclude_deprecated") and $options.exclude_deprecated == true) then
+        map(select(.id | endswith(".deprecated") | not))
+      else
+        .
+      end
+    | map(. + {
+        event_namespace: (if .id | index(".") then (.id | split(".") | .[0:-1] | join(".")) else "other" end)
```

Comment:
we should never use group id in JQ - please use full event name.

Reply:
This was copied from the existing (or previous) jq query on this, as far as I understand the `id` here is the resolved `id` (derived from the name and prefix, not the `id` from the registry entry.
But I'm going to be moving this to be isolated to weaver for testing anyway.

Reply:
All changes to the semconv.jq have been reverted.